backend/server.js

```diff
@@ -248,19 +251,19 @@ async function getSearch() {
       return null;
     }
 
-    return search.create(
-      employeeMap, employeesSearchIndex,
-      [{
-        searchIndex: springSearchIndex,
-        termDump: springData,
-        termId: '201830',
-      },
-      {
-        searchIndex: fallSearchIndex,
-        termDump: fallData,
-        termId: '201810',
-      }],
-    );
+    const dataLib = DataLib.loadData({
```

Comment:
Definitely want to be able to dynamically load in any semester that the scrapers find on Northeastern's site so we don't have to continuously change things every semester. The hardcoded paths in `server.js` are just for now, and totally want to switch them to dynamic in the future. When we do this, we would also have to update the hardcoded dropdown in `home.js` in the frontend. Opened #36 to keep track of the progress of this. 

Not sure about hardcoding that `10` to be fall and `30` to be spring though. These endings are just made up by Northeastern and could totally change in the future. Additionally, other colleges use different endings which would be a nuisance to keep track of. 

On a side note, these termIds don't really make sense either. The termId for Fall 2017 is `201810`, which you would think would be in 2018 because it starts with 2018. The termId for Spring of 2018 is 201830 which makes sense. Some of the other termIds arn't consistent either. On top of that, other colleges have different termIds inconsistencies too.

Reply:
Perhaps their thought process is like this:
"Academic year 2017-2018"
truncate to last year:
"2018"
Let fall = 10
winter = 20
spring = 30
summer = 40

2017 Fall = "201810"
2017 Winter = "201820"
2018 Spring = "201830"
2018 Summer = "201840"

Reply:
Oh that could totally be it. :P 

When we do want to spend some time on this I'm down to make it totally dynamic based on what the scrapers find.

Reply:
Do you have historical data? If we could find a pattern then it would be pretty easy to implement.

Reply:
https://gist.github.com/ryanhugh/4bb105f186f36032fcd7d4f38c1fa6c1

Reply:
Looks like there is a pattern! But it would be different for other schools, if we ever do want to expand. I'l get some data from another school

Reply:
Here's the data for GTech https://gist.github.com/ryanhugh/7e8b285bd8443e5a10b94db5e7d72359
If you want to look at some others look at the source of the page of some of these URLS:  https://github.com/ryanhugh/searchneu/blob/master/backend/scrapers/classes/differentCollegeUrls.js